editor/plugins/style_box_editor_plugin.cpp

```diff
@@ -112,6 +112,7 @@ StyleBoxPreview::StyleBoxPreview() {
 	grid_preview = memnew(Button);
 	// This theme variation works better than the normal theme because there's no focus highlight.
 	grid_preview->set_theme_type_variation("PreviewLightButton");
+	grid_preview->set_tooltip_text(TTRC("Toggle margins preview grid."));
```

Comment:
I'm wondering why we use `TTRC()` here instead of `TTR()`.

Reply:
TTRC is nowadays preferred. TTR translates in place, TTRC does nothing, which enables auto-translation and easier language switching.